WTraining/app/controllers/user.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const userService = require('../services/user');
+const logger = require('../logger');
+
+const createUser = (req, res, next) =>
+  userService
+    .createUser(req.body)
+    .then(user => {
+      delete user.dataValues.password;
```

Comment:
try to the values are inmutable. I recommend you to use serializers and mappers when you need to transform data